### ISSUE TITLE
Add Social Anxiety Term

### ISSUE THREAD

**Issue by akyocum** [obophenotype/human-phenotype-ontology#10167]
**Preferred term label:**
Social Anxiety

**Synonyms**
Social Phobia

**Definition (free text, please give PubMed ID)**
An intense, persistent fear of being watched and judged by others

**Parent term (use hpo.jax.org/app)**
Anxiety:  HP:0000739

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000029
- label: Social Anxiety
- parents: [Anxiety (HP:0000739)](https://hpo.jax.org/app/browse/term/HP:0000739)
- pmid: 
- def: An intense, persistent fear of being watched and judged by others.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10167
- synonyms:

**Comment by bcoleman96** [comment]
I would suggest the following revisions:

ID: HP:6000029
label: Social Anxiety
parents: [Anxiety (HP:0000739)](https://hpo.jax.org/app/browse/term/HP:0000739),  [Cognitive distortion (HP:5200242)](https://hpo.jax.org/app/browse/term/HP:5200242)
pmid:
def: A persistent, irrational fear of being negatively judged by others because of one's social behavior or general presentation.
comment:
orcid: orcid:0000-0002-0736-9199
github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10167
synonyms:

**Comment by pnrobinson** [comment]
Added this definition
Excessive fear of embarrassment, humiliation, or rejection when exposed to possible negative evaluation by others when engaged in a public performance or social interaction.
from here -- https://www.ncbi.nlm.nih.gov/books/NBK555890/
and added second parentage
